src/color.rs

```diff
@@ -16,23 +16,29 @@ fn str_to_angle(s: impl AsRef<str>) -> f64 {
     f64::from(checksum) / 65536.0 * 360.0
 }
 
-/// Converts an identifier to RGB color.
+/// Converts RGB tuple to a 24-bit number.
 ///
 /// Returns a 24-bit number with 8 least significant bits corresponding to the blue color and 8
 /// most significant bits corresponding to the red color.
+fn rgb_to_u32((r, g, b): (f64, f64, f64)) -> u32 {
```

Comment:
why use a tuple instead of three standalone argumetns?

Reply:
Because `hsluv_to_rgb` returns a tuple, so I wanted to just past it through in `str_to_color`.